src/libs/Navigation/AppNavigator/AuthScreens.js

```diff
@@ -368,6 +369,12 @@ class AuthScreens extends React.Component {
                     component={IOUSendModalStackNavigator}
                     listeners={modalScreenListeners}
                 />
+                <RootStack.Screen
+                    name="OAUTH_PLAID_REDIRECT"
+                    options={modalScreenOptions}
+                    component={PlaidOAuthModalStackNavigator}
```

Comment:
NAB, but curious if there is anyway to avoid adding another modal stack nav e.g. hooking into an existing stack.